### PR TITLE
feat: mirrorxyz data provider

### PR DESCRIPTION
**Pull request by sampolgar**
Mirrorxyz data provider as per https://github.com/sismo-core/sismo-hub/issues/1444

### CODE REVIEW

File: group-generators/helpers/data-providers/mirrorxyz/interface-schema.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  "functions": [
+    {
+      "name": "Get Post Collector",
+      "functionName": "getPostCollectors",
+      "countFunctionName": "getPostCollectorsCount",
+      "description": "Returns post collectors for a given post",
+      "args": [
+        {
+          "name": "Contract Address",
+          "argName": "address",
```
Comment (MartinGbz) [inline group-generators/helpers/data-providers/mirrorxyz/interface-schema.json:14]: Regarding your `QueryMirrorXyzInput` type, argName is `contract` here

**Review by MartinGbz — COMMENTED** [review]
@sampolgar added 2 comments here

File: group-generators/helpers/data-providers/mirrorxyz/interface-schema.json (hunk 1 above)
Comment (sampolgar) [reply group-generators/helpers/data-providers/mirrorxyz/interface-schema.json]: Thanks!

File: group-generators/helpers/data-providers/mirrorxyz/index.ts [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      url:
+        url ??
+        "https://api.thegraph.com/subgraphs/name/detoo/mirror-xyz-optimism",
+    });
+  }
+
+  public async getPostCollectors({
+    contract,
+  }: QueryMirrorXyzInput): Promise<FetchedData> {
+    const query = gql`
```
[the remark this answers is not in the document]
Comment (sampolgar) [reply group-generators/helpers/data-providers/mirrorxyz/index.ts]: Beautiful, thanks, I've added it

**Comment by sampolgar** [comment]
Thanks @MartinGbz!

**Pull request merged**